orca-clouddriver/src/test/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/providers/aws/cloudformation/CloudFormationForceCacheRefreshTaskSpec.groovy

```diff
@@ -36,4 +37,29 @@ class CloudFormationForceCacheRefreshTaskSpec extends Specification {
     then:
     1 * task.cacheService.forceCacheUpdate('aws', CloudFormationForceCacheRefreshTask.REFRESH_TYPE, Collections.emptyMap())
   }
+
+  @Unroll
+  void "should add scoping data if available"() {
+    setup:
+    task.cacheService = Mock(CloudDriverCacheService)
+    def stage = stage()
+    stage.context.put("credentials", credentials)
+    stage.context.put("regions", regions)
+
+
+    when:
+    task.execute(stage)
+
+    then:
+    1 * task.cacheService.forceCacheUpdate('aws', CloudFormationForceCacheRefreshTask.REFRESH_TYPE, data)
+
+    where:
+    credentials   | regions       | data
```

Comment:
minor nit ... expected sides of a spock table should be prefixed with `||` (vs. `|`)

```
credentials      | regions      || expectedData
```